dom/interfaces.html

```diff
@@ -18,11 +18,12 @@ <h1>DOM IDL tests</h1>
 
 var idlArray = new IdlArray();
 
-function doTest([html, dom]) {
+function doTest([html, dom, uievents]) {
   // HTML is needed for EventHandler. Provide a dummy interface for
   // LinkStyle which HTML depends on but we're not testing.
   idlArray.add_untested_idls('interface LinkStyle {};');
   idlArray.add_untested_idls(html);
```

Comment:
I was wondering if using `add_dependency_idls` *here* would do the trick, and avoid 'interface LinkStyle {};' and the extra bits from uievents entirely.

Reply:
Oh, that makes more sense. That does indeed work.